src/main/java/com/khutircraftubackend/exception/UnknownExceptionResolver.java

```diff
@@ -0,0 +1,57 @@
+package com.khutircraftubackend.exception;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+import org.springframework.web.servlet.HandlerExceptionResolver;
+import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.servlet.view.json.MappingJackson2JsonView;
+
+@Slf4j
+@Component
+@SuppressWarnings("NullableProblems")
+public class UnknownExceptionResolver implements HandlerExceptionResolver {
+
+    private final MappingJackson2JsonView jsonView;
+    private static final HttpStatus STATUS = HttpStatus.INTERNAL_SERVER_ERROR;
+
+    public UnknownExceptionResolver() {
+        this.jsonView = new MappingJackson2JsonView();
+        this.jsonView.setExtractValueFromSingleKeyModel(true);
+    }
```

Comment:
better keep constructor for spring framework 
and use static block here: 

```suggestion
    private static final MappingJackson2JsonView jsonView;
    private static final HttpStatus STATUS = HttpStatus.INTERNAL_SERVER_ERROR;

    static {
        jsonView = new MappingJackson2JsonView();
        jsonView.setExtractValueFromSingleKeyModel(true);
    }

```

and if you do so, then change line 51 to : 
```java
ModelAndView mv = new ModelAndView(jsonView);
```